.gitignore

```diff
@@ -28,3 +34,6 @@ logs
 .DS_Store
 
 out/jreleaser/
+
+# Test files
+/dropwizard-migrations/src/test/resources/test-db.mv.db
```

Comment:
Could you please move this into a `.gitignore` file inside `dropwizard-migrations`?